src/main/java/uk/gov/homeoffice/digital/sas/timecard/model/KafkaEventMessage.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.homeoffice.digital.sas.timecard.model;
+
+import javax.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import uk.gov.homeoffice.digital.sas.timecard.enums.KafkaAction;
+
+@AllArgsConstructor
+@Getter
+@Setter
+@Builder
+public class KafkaEventMessage {
+
+  @NotNull
+  private TimeEntry resource;
```

Comment:
Can this be renamed to timeEntryResource rather than a generic resource

Reply:
We are now using generic types in this class, so I have left the resource name as it is